### PR TITLE
Changes Manager Guide to Feature Guide, updates section for 2.1

### LINKED ISSUE #313: Cleanup front matter on manager_guide pages
### Descriptive summary of request

apply proper `a-z`, `tags`, `categories`, and `keywords` to pages in the manager_guide directory

### PR DESCRIPTION
**Pull request by chrisdaaz**
This PR addess several issues identified by the repo managers group, related to the 2.1 release. 

fixes #313, #303, #300, #297, #296, #295, #294, #293, #292,  and #291.

I'm asking @julesies to review the content as a first step before merging.

### CODE REVIEW

File: pages/samvera/manager_guide/hyrax_2.1/batch-ops-2.1.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+  - label: 'Hyrax 1.0'
+    link:  'batch-works-1.0.html'
+  - label: 'Hyrax 2.0'
+    link:  'batch-works-2.0.html'
+  - label: 'Hyrax 2.1'
+    link:  'batch-ops-2.1.html'
+    selected: 'true'
+---
+## Uploading Works
+Batch upload lets you upload multiple files at once as separate works. This is useful when you have several works, where each work is a single file, and each work has the same creator, such as technical reports from a organization or photographs by an artist.
```
Comment (julesies) [inline pages/samvera/manager_guide/hyrax_2.1/batch-ops-2.1.md]: can we change this to say "...and each work has the same descriptive metadata, such as technical reports from a organization or photographs by an artist."

File: pages/samvera/manager_guide/hyrax_2.1/batch-ops-2.1.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+To create works as a batch, go to your Dashboard and click on `Works` in the sidebar. There, you will see a table of your works and options to `Add new work` or `Create batch of works`.
+
+![Create batch of works](/images/screenshots/create-batch-works-1.png)
+
+If you have more than one work type, you will be prompted to select the work type. You can only create a batch of works of the same work type.
```
Comment (julesies) [inline pages/samvera/manager_guide/hyrax_2.1/batch-ops-2.1.md]: add "There is a limit of 100 works per batch."

File: pages/samvera/manager_guide/hyrax_2.1/batch-ops-2.1.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+### Adding Files to Works
+The “Files” tab is for uploading files, either locally or from a cloud service. This works in the same way as adding files to a new work. You can upload one file at a time, or select multiple files.
```
Comment (julesies) [inline pages/samvera/manager_guide/hyrax_2.1/batch-ops-2.1.md]: Add limit of 100 here too?

File: pages/samvera/manager_guide/hyrax_2.1/batch-ops-2.1.md [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+
+## Deleting Works
+As a Repository Administrator or the owner of works, you have the ability to delete works in bulk. From your `Dashboard`, click on `Works`, in the sidebar. To the left of each collection title, there is a box that you can click on to select that work. Whenever you select a box, whether it is one or more or all, a red button will appear labelled `Delete Selected`. Clicking the button will initiate a dialog box asking if you are sure. This step is irreversible.
+
+## Deleting Collections
+As a Repository Administrator, you have the ability to delete collections in bulk. From your `Dashboard`, click on `Collections`, in the sidebar. To the left of each collection title, there is a box that you can click on to select that collection. Whenever you select a box, whether it is one or more or all, a red button will appear labelled `Delete collections`. Clicking the button will initiate a dialog box asking if you are sure. This step is irreversible.
```
Comment (julesies) [inline pages/samvera/manager_guide/hyrax_2.1/batch-ops-2.1.md]: Add "You can not delete Admin Sets that contain. You must first delete the works or move them to another set".

File: pages/samvera/manager_guide/hyrax_2.1/collections-2.1.md [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+version:
+  id: 'hyrax_2.1-stable'
+  versions:  
+  - label: 'Hyrax 2.0'
+    link:  'collections-2.0.html'
+  - label: 'Hyrax 2.1'
+    link:  'collections-2.1.html'
+    selected: 'true'
+---
+Hyrax 2.1 comes with a suite of new options for creating and configuring collection types to satisfy a number of use cases for cultural heritage and research institutions. This page focuses on the basics of an end user creating a collection. For a detailed overview of collection types and configurations, please visit the [Collections and Admin Sets](http://samvera.github.io/collection-overview.html) section of this website.
```
Comment (julesies) [inline pages/samvera/manager_guide/hyrax_2.1/collections-2.1.md]: Let's add "Two types of User Collections come with Hyrax - User Collections and Admin Sets"?

File: pages/samvera/manager_guide/hyrax_2.1/collections-2.1.md [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+
+## Create a User Collection
+
+Login to the repository, navigate to the Dashboard, click on Collections in the sidebar:
+
+![Collections Dashboard](/images/screenshots/collections-2.1.png)
+
+Click on the `New Collection` button at the top of the screen. If the repository contains more than one collection type, then a dialog box will pop up, giving you the option to select a collection type for the new collection.
```
Comment (julesies) [inline pages/samvera/manager_guide/hyrax_2.1/collections-2.1.md]: add "Some collection types are not available to all users of the system. Users will only see choices for collection types they have permissions to create" ???

File: pages/samvera/manager_guide/hyrax_2.1/collections-2.1.md [hunk 7]
```diff
@@ -0,0 +1,57 @@
...
+
+## Adding Works to Collections
+
+There are several ways to add works to collections:
+
+- From the Dashboard, click “Works” and select one or more works. Click “Add to Collection”. To add the works to a new collection, click “Add to new Collection” at the bottom of the dialog box. To add the works to an existing collection, select the collection in the dialog box, then click “Save Changes.”
+- In the Browse view of a work, click “Edit” and select a collection in the “Relationships” tab. Click “Add.”  To select more than one collection, hold Control while selecting.
+- When creating a new work, select a collection in the “Relationships” tab.
+
+The visibility of a collection does not affect the visibility of a work. For example, if an Open Access work is added to a Private collection, the work will remain Open Access, but a link to the collection itself can not be discovered.
```
Comment (julesies) [inline pages/samvera/manager_guide/hyrax_2.1/collections-2.1.md]: NICE

File: pages/samvera/manager_guide/hyrax_2.1/collections-2.1.md [hunk 8]
```diff
@@ -0,0 +1,57 @@
...
+
+## Editing Collections
+
+- To edit metadata for a collection, click on the “Collections” link in the Dashboard and click on the collection.  Click the “Edit” button.
+- To change collection visibility, click on the “Collections” link in the Dashboard and click on the collection. Click the “Edit” button. Select the “Discovery” tab at the top of the page.  
+- Collections can be either public, limited to institution only, or private.
+- To remove works from a collection, click on the “Collections” link in the Dashboard and click on the collection. Navigate to the “Works” heading. Click “Remove” next to the work that you would like to delete.
+- To delete a collection, click on the “Collections” link in the Dashboard.  Select “Delete Collection” from the “Actions” dropdown menu next to the collection that you would like to delete.
```
Comment (julesies) [inline pages/samvera/manager_guide/hyrax_2.1/collections-2.1.md]: Add "deleting collections, do not have any effect on the works that are associated with the collections" ?

File: pages/samvera/manager_guide/hyrax_2.1/configuration-2.1.md [hunk 9]
```diff
@@ -0,0 +1,89 @@
...
+
+![Collection Types](images\screenshots\admin-configuration-collection-types.png)
+
+Repository Administrators can create and define collection types for the repository. Collection types include Administrative Sets and User Collections. Detailed information about collections and collection types are available in the [Collections and Admin Sets](http://samvera.github.io/collection-overview.html) section of this website.
+
+## Settings: Pages
+
+![Pages](images\screenshots\admin-configuration-pages.png)
+
+The Pages panel allows administrators to edit the About Page, Help Page, Deposit Agreement, and Terms of Use. These are static pages in the repository. This is not quite a [WYSIWYG editor](https://en.wikipedia.org/wiki/WYSIWYG). The text will conform to the CSS styles used by your repository. Tip: to add hyperlinks, create the links in a rich text editor, like Microsoft Word, and paste the link into the Pages text box. There is currently no way to create a link directly in the text editor.
```
Comment (julesies) [inline pages/samvera/manager_guide/hyrax_2.1/configuration-2.1.md]: nice!

File: pages/samvera/manager_guide/hyrax_2.1/configuration-2.1.md [hunk 10]
```diff
@@ -0,0 +1,89 @@
...
+
+There are several features that are automatically enabled in Hyrax, but these can be toggled on and off from the Features panel:
+
+![Configuration Features](images\screenshots\admin-configuration-features-2.1.png)
+
+The highlighted option in the Actions column signifies the current selection. Click on the Off option for any feature you wish to disable for your repository. Here is a [complete list](https://github.com/samvera/hyrax/wiki/Feature-matrix) of features in Hyrax.
+
+## Workflow Roles
+
+The Workflow Roles panel will allow repository administrators to search for any registered user in the repository and assign them a role in an Administrative Set workflow. From the page, you can Assign Roles or view Current Roles.
```
Comment (julesies) [inline pages/samvera/manager_guide/hyrax_2.1/configuration-2.1.md]: so this i think is only needed if people have "custom workflows" i'm not sure if it's recommended to add roles this way if you are using the "shipped" workflows.

File: pages/samvera/manager_guide/hyrax_2.1/create-works-2.1.md [hunk 11]
```diff
@@ -0,0 +1,61 @@
...
+
+## Description (required)
+The “Description” tab is for adding descriptive metadata. Fill out the fields marked “Required”. Clicking “Additional Fields” will show more metadata fields, which are optional. Customizing fields is possible by creating a new Work Type.
+
+## Files (required)
+![Add New Generic Work | Files](images\screenshots\user-addnewgenericwork-files-2.png)
+
+The “Files” tab is for uploading files, either locally or from a cloud service (e.g. DropBox, Google Drive, Box.com, etc.). You can browse local files to upload, or drag and drop files from your desktop. If your repository is configured to interact with a cloud service, users can click “Browse cloud files” and securely connect to their own Dropbox. They can then browse cloud files to upload. Files can also be removed from the deposit in the “Files” tab.
```
Comment (julesies) [inline pages/samvera/manager_guide/hyrax_2.1/create-works-2.1.md]: Should we add "Repositories can be set up for metadata-only works, in this case files are not required."

File: pages/samvera/manager_guide/hyrax_2.1/create-works-2.1.md [hunk 12]
```diff
@@ -0,0 +1,61 @@
...
+
+## Relationships (optional)
+![Add New Generic Work | Relationships](images\screenshots\user-addnewgenericwork-relationships.png)
+
+The “Relationships” tab lets you add the work to an Administrative Set or a Collection. Administrative users can add a work to any Administrative Set or Collection in the repository. Regular users are only able to add works to Collections that they have created or collections for which they have been granted a depositor/manager role. The Administrative Set and Collections fields will be automaticall populated with options to which the user has depositor access.
```
Comment (julesies) [inline pages/samvera/manager_guide/hyrax_2.1/create-works-2.1.md]: we need to add a whole section here about child works, parent works........is this the best place for it?

Comment (julesies) [reply pages/samvera/manager_guide/hyrax_2.1/create-works-2.1.md]: typo here: fields will be automaticall

File: pages/samvera/manager_guide/hyrax_2.1/create-works-2.1.md [hunk 13]
```diff
@@ -0,0 +1,61 @@
...
+
+## Visibility
+You can select one of five options for visibility: Public, Your Institution, Embargo, Lease, Private. If you are not seeing five visibility options, it is likely that your work is set to deposit to an administrative set with fewer visibility options than the default. Visibility only controls who can view or download your work -- it does not control edit access.
```
Comment (julesies) [inline pages/samvera/manager_guide/hyrax_2.1/create-works-2.1.md]: Add "Options for visibility can be restricted if the Admin Set has applied restrictions."

File: pages/samvera/manager_guide/hyrax_2.1/create-works-2.1.md [hunk 14]
```diff
@@ -0,0 +1,61 @@
...
+- Institution restricts access to works and work metadata to users with login privileges. Users will need to be logged into the repostory to access the work.
+- Embargo lets you restrict access to Private or Institution until a specified date, when it will be opened to the public or your institution
+- Lease permits access to the work to the public or Institution until a specified date, when it will be restricted to Private or your institution
+
+## Deposit Agreement (required)
+The depositor must check the deposit agreement checkbox in order to deposit. The agreement can be customized by repository administrators by going to the Dashboard, and editing the Deposit Agreement from the Pages panel.
+
+## Depositing as a proxy
+Users can designate other users of the system to deposit on their behalf by visiting their dashboard. Once someone has been set as a proxy for another user, the sidebar will offer the option to deposit on behalf of yourself or the other user. If you deposit on behalf of another user, the ownership of the work will automatically be transferred to that user when the deposit completes.
```
Comment (julesies) [inline pages/samvera/manager_guide/hyrax_2.1/create-works-2.1.md]: add "Proxies will retain editing privileges to works they deposit unless the owner revokes access"

File: pages/samvera/manager_guide/hyrax_2.1/edit-works-2.1.md [hunk 15]
```diff
@@ -0,0 +1,76 @@
...
+
+In the edit view for a work, you can change the description, add files, edit relationships, edit sharing, and edit visibility. If you edit any of these settings, you must click “Save” in order to apply your changes. Note that if you add files, you must recheck the deposit agreement before saving.
+
+If you edit the sharing or visibility of the work, you can either opt to have all files in the work inherit these permissions, or you can opt to have the individual files retain their permissions so that they can be edited manually.
+
+Set child/parent work: In the “Relationships” tab, you can now add a child or parent work to the work - this is only available after the work is deposited, not at the time of deposit. A child work added to this work will appear as an Item in this work. A parent work will add this work to its Items. To add a child or parent work, enter the unique ID (found at ???) of the work under “Child Work” or “Parent Work” and click “Add”. Click “Save” to apply your changes.
```
Comment (julesies) [inline pages/samvera/manager_guide/hyrax_2.1/edit-works-2.1.md]: ohhh can you only do this on edit? i don't think you do the unique ID, it's a title search. or you "start on the parent attach a child"

File: pages/samvera/manager_guide/hyrax_2.1/edit-works-2.1.md [hunk 16]
```diff
@@ -0,0 +1,76 @@
...
+
+## Editing a work - file manager
+To change the order of files in a work or set a representative file, you have to use the File Manager. In the Browse view of a work, click “File Manager”.
```
Comment (julesies) [inline pages/samvera/manager_guide/hyrax_2.1/edit-works-2.1.md]: this should be followed by a frowny face ;)

File: pages/samvera/manager_guide/hyrax_2.1/lease-embargoes-2.1.md [hunk 17]
```diff
@@ -0,0 +1,59 @@
...
+
+The Embargo management page contains three tabs:
+- All Active Embargoes: Works where embargoes are current
+- Expired Active Embargoes: Works where embargo dates have passed but they are still under embargo status
+- Deactivated Embargoes: Works that are currently out of embargo status and access status has been changed
+
+To deactivate embargoes:
+- Click the “Expired Active Embargoes” tab
+- Select works to release (optionally leave all files in embargo state by unchecking the box “Change all files within work __ to _ status _ )
+- Click button “Deactivate Embargoes for Selected” or deactivate individually
```
Comment (julesies) [inline pages/samvera/manager_guide/hyrax_2.1/lease-embargoes-2.1.md]: wonder should we add anything about how deactivating an embargo before the original date is passed does not lift the embargo. You must manually change the visibility to the desired state.

File: pages/samvera/manager_guide/hyrax_2.1/batch-ops-2.1.md [hunk 18]
```diff
@@ -0,0 +1,54 @@
...
+### Adding Files to Works
+The “Files” tab is for uploading files, either locally or from a cloud service. This works in the same way as adding files to a new work. You can upload one file at a time, or select multiple files.
+
+![Upload files](/images/screenshots/create-batch-works-2.png)
+
+After uploading files, you can edit the `Display Label` and `Resource Type` for each Work. Each work can be assigned its own `Display Label` (i.e. Title) and `Resource Type`, but all other metadata will be applied to all works created in the batch. To change the resource type of all files at once, select the desired resource type for the first file and click “Set all to this Resource Type”
+
+![Editing Display Labels and Resource Types](/images/screenshots/create-batch-works-3.png)
+
+### Describing Works
```
Comment (julesies) [inline pages/samvera/manager_guide/hyrax_2.1/batch-ops-2.1.md]: is this really about batch editing MD for existing works? or when batch loading.

File: pages/samvera/manager_guide/hyrax_2.1/batch-ops-2.1.md (hunk 1 above)
Comment (chrisdaaz) [reply pages/samvera/manager_guide/hyrax_2.1/batch-ops-2.1.md]: yep

File: pages/samvera/manager_guide/hyrax_2.1/batch-ops-2.1.md [hunk 19]
```diff
@@ -0,0 +1,54 @@
...
+The `Descriptions` tab asks for `Creator`, `Keyword`, and `Rights Statement`. You also have the option to apply additional description fields. Each description field you apply will be applied to all works in the batch.
+
+![Descriptions](/images/screenshots/create-batch-works-4.png)
+
+### Relationships, Sharing, and Saving
+The  `Relationships` and `Sharing` tabs are identical to the tabs you would see while depositing a work. The process for Saving the batch of works is the same as saving a single work. Every selection you make for these options will be applied to all of the works in the batch.
+
+## Adding Works to a Collection
+As a Repository Administrator or the owner of works, you have the ability to add one or more works to a collection. From your `Dashboard`, click on `Works`, in the sidebar. To the left of each collection title, there is a box that you can click on to select that work. Whenever you select a box, whether it is one or more or all, a red button will appear labelled `Add to collection`. Clicking the button will initiate a dialog box asking if you are sure. This step is irreversible.
```
[the remark this answers is not in the document]
Comment (chrisdaaz) [reply pages/samvera/manager_guide/hyrax_2.1/batch-ops-2.1.md]: deleted the "irreversible" sentence and added the other method of adding works to a collection.

File: pages/samvera/manager_guide/hyrax_2.1/batch-ops-2.1.md (hunk 4 above)
Comment (chrisdaaz) [reply pages/samvera/manager_guide/hyrax_2.1/batch-ops-2.1.md]: done

File: pages/samvera/manager_guide/hyrax_2.1/collections-2.1.md (hunk 5 above)
Comment (chrisdaaz) [reply pages/samvera/manager_guide/hyrax_2.1/collections-2.1.md]: done

File: pages/samvera/manager_guide/hyrax_2.1/collections-2.1.md (hunk 6 above)
Comment (chrisdaaz) [reply pages/samvera/manager_guide/hyrax_2.1/collections-2.1.md]: done

File: pages/samvera/manager_guide/hyrax_2.1/create-works-2.1.md (hunk 11 above)
Comment (chrisdaaz) [reply pages/samvera/manager_guide/hyrax_2.1/create-works-2.1.md]: done

File: pages/samvera/manager_guide/hyrax_2.1/create-works-2.1.md (hunk 12 above)
Comment (chrisdaaz) [reply pages/samvera/manager_guide/hyrax_2.1/create-works-2.1.md]: fixed the typo, we should open an issue (if we don't already have one) about documentation to add about child and parent works. shouldn't be a blocker for this PR

File: pages/samvera/manager_guide/hyrax_2.1/create-works-2.1.md (hunk 13 above)
Comment (chrisdaaz) [reply pages/samvera/manager_guide/hyrax_2.1/create-works-2.1.md]: done

File: pages/samvera/manager_guide/hyrax_2.1/create-works-2.1.md (hunk 14 above)
Comment (chrisdaaz) [reply pages/samvera/manager_guide/hyrax_2.1/create-works-2.1.md]: done.

File: pages/samvera/manager_guide/hyrax_2.1/edit-works-2.1.md (hunk 15 above)
Comment (chrisdaaz) [reply pages/samvera/manager_guide/hyrax_2.1/edit-works-2.1.md]: changed it to title

File: pages/samvera/manager_guide/hyrax_2.1/edit-works-2.1.md (hunk 16 above)
Comment (chrisdaaz) [reply pages/samvera/manager_guide/hyrax_2.1/edit-works-2.1.md]: haha

File: pages/samvera/manager_guide/hyrax_2.1/batch-ops-2.1.md (hunk 18 above)
Comment (chrisdaaz) [reply pages/samvera/manager_guide/hyrax_2.1/batch-ops-2.1.md]: changed alt text to clarify

**Pull request merged**